src/ReactUnipika/__tests__/ReactUnipika.visual.test.tsx

```diff
@@ -92,47 +92,46 @@ test('ReactUnipika: json with container size collapsed initially', async ({
     await expectScreenshot({component: page});
 });
 
-test('ReactUnipika: with case insensitive search with matches', async ({
+test('ReactUnipika: search in collapsed - collapsed tree with search', async ({
     mount,
     expectScreenshot,
     page,
 }) => {
-    await mount(<Stories.WithCaseInsensitiveSearch />, {width: 1280});
+    await mount(<Stories.Json />, {width: 1280});
 
-    await page.getByTestId('qa:structuredyson:search').locator('input').fill('Attr');
-
-    await expectScreenshot({component: page});
-});
-test('ReactUnipika: with case insensitive search with matches 2', async ({
-    mount,
-    expectScreenshot,
-    page,
-}) => {
-    await mount(<Stories.WithCaseInsensitiveSearch />, {width: 1280});
+    // Collapse all
+    await page.getByTestId('qa:structuredyson:collapse-all').click();
 
+    // Enter search term
     await page.getByTestId('qa:structuredyson:search').locator('input').fill('attr');
 
-    await expectScreenshot({component: page});
-});
-test('ReactUnipika: with case sensitive search no matches', async ({
-    mount,
-    expectScreenshot,
-    page,
-}) => {
-    await mount(<Stories.Json />, {width: 1280});
-
-    await page.getByTestId('qa:structuredyson:search').locator('input').fill('Attr');
+    // Wait for search to complete
+    await page.waitForTimeout(300);
```

Comment:
Isn't it better to wait for some search results like a highlighted text or `N matches`?